Userland/Libraries/LibWeb/Layout/SVGFormattingContext.cpp

```diff
@@ -273,7 +275,7 @@ void SVGFormattingContext::run(Box const& box, LayoutMode layout_mode, Available
     // 5.2. Grouping: the ‘g’ element
     // The ‘g’ element is a container element for grouping together related graphics elements.
     box.for_each_in_subtree_of_type<Box>([&](Box const& descendant) {
-        if (is<SVGGraphicsBox>(descendant) && !is<SVGGeometryBox>(descendant) && !is<SVGTextBox>(descendant)) {
+        if (is_container_element(descendant)) {
             auto const& svg_graphics_box = static_cast<SVGGraphicsBox const&>(descendant);
```

Comment:
Looks like this is not safe anymore (from macOS CI):

```
/Users/runner/work/1/s/Userland/Libraries/LibWeb/Layout/SVGFormattingContext.cpp:279:44: runtime error: downcast of address 0x62d00048d840 which does not point to an object of type 'const Web::Layout::SVGGraphicsBox'
0x62d00048d840: note: object is of type 'Web::Layout::Box'
 00 00 00 00  f0 ad f9 12 01 00 00 00  00 00 00 00 00 00 00 00  40 cc 48 00 d0 62 00 00  40 e4 48 00
              ^~~~~~~~~~~~~~~~~~~~~~~
              vptr for 'Web::Layout::Box'
```

I don't think we need the `static_cast` here at all though? We're just using `descendant` as a key to fetch the `LayoutState::UsedValues`.

Reply:
Yep, the cast is unnecessary. It looks like `<symbol>` elements just use a plain `Layout::Box` (which is a little surprising as they are graphics elements, but it's alright for now).